Userland/Libraries/LibAudio/GenericTypes.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, Lucas Chollet <lucas.chollet@free.fr>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <AK/Vector.h>
+
+namespace Audio {
+
+enum class FlacPictureType : u32;
```

Comment:
Please rename this type to ID3PictureType and move the definition into the GenericTypes header as well. You can leave a comment that FLAC's picture type as defined in section 11.20 (in draft 3; the numbers have changed with draft 5 but as I said I'll address that once the spec is finalized) is the same as the ID3 picture type.